packages/api/src/controllers/playback.ts

```diff
@@ -47,7 +47,8 @@ const getAssetPlaybackUrl = async (
   cid: boolean
 ) => {
   const asset = cid
-    ? await db.asset.getByIpfsCid(id)
+    ? (await db.asset.getByIpfsCid(id)) ??
+      (await db.asset.getBySourceURL("ipfs://" + id))
```

Comment:
might wanna add the `ar://` here if we add support for that on task runner

Reply:
Added in https://github.com/livepeer/studio/pull/1372